src/main/resources/vm/script/datamanagement/uploadFunctions.vm

```diff
@@ -89,76 +87,45 @@ function uploadLfnFile {
         chooseRandomSE
         DEST=${RESULT}
     fi
-    done=0
+    local done=0
     while [ $nrep -gt $done ] && [ "${DEST}" != "" ]
     do
         if [ "${done}" = "0" ]
         then
-            lcg-del -v -a ${OPTS} lfn:${LFN} &>/dev/null
-            lfc-ls ${LFN}
-            if [ $? = 0 ]
-            then
-                lfc-rename ${LFN} ${LFN}-garbage-${HOSTNAME}-${PWD}
-            fi
-            flag="cr"
-            lfc-mkdir -p `dirname ${LFN}`
-            lcg-cr -v ${OPTS} -d ${DEST} -l lfn:${LFN} file:${FILE} &> lcg-log
+            dirac-dms-remove-files ${OPTS} ${LFN} &>/dev/null
+            (time dirac-dms-add-file ${OPTS} ${LFN} ${FILE} ${DEST}) &> dirac.log
+            local command="dirac-dms-add-file"
+            local source=$(hostname)
```

Comment:
The commands after the dirac upload command causes that the result status "$?" is lost